### PR TITLE
V0.4.0

### PR DESCRIPTION
**Pull request by chapmandu2**

### CODE REVIEW

File: R/create_realistic_org.R [hunk 1]
```diff
@@ -67,7 +67,7 @@ create_realistic_org <- function(n_children = 4, max_depth = 3, prob=0.3, .f=NUL
   }
 
   out <- create_regular_org(n_children, max_depth) %>%
-    tidygraph::mutate(unit_id = dplyr::row_number(),
+    tidygraph::mutate(unit_id = dplyr::row_number() %>% as.character(),
```
Comment (mammykins) [inline R/create_realistic_org.R]: why as.character? int comparison will be faster; is it for plotting reasons?

File: R/get_df_format.R [hunk 2]
```diff
@@ -39,8 +39,8 @@ get_df_format <- function(df) {
       return(format_id)
     } else if (identical(names(template_df_format), names(query_df_format)) &
                !identical(unname(template_df_format), unname(query_df_format))) {
-      stop(sprintf('%s format but column types incorrect, try data("%s") for an example', format_id, template_df))
-      #return(format_id)
+      warning(sprintf('%s format but column types incorrect, try data("%s") for an example', format_id, template_df))
```
Comment (mammykins) [inline R/get_df_format.R]: if data type is wrong will downstream work?

File: R/create_realistic_org.R (hunk 1 above)
Comment (chapmandu2) [reply R/create_realistic_org.R]: I realised that unit id's will often not be integers - eg in the people survey they take the form DWP0123.

File: R/get_df_format.R (hunk 2 above)
Comment (chapmandu2) [reply R/get_df_format.R]: Maybe not, but I felt that warning rather than stopping was legitimate and less of a pain!

**Comment by chapmandu2** [comment]
Thanks for the comments.  I'll merge but put the suggestions down as issues/enhancements.

**Pull request merged**